src/components/proposal/ProposalTableInstrumentScientist.tsx

```diff
@@ -121,14 +119,24 @@ const ProposalTableInstrumentScientist: React.FC<{
     ...DefaultQueryParams,
     call: NumberParam,
     instrument: NumberParam,
-    proposalStatus: withDefault(NumberParam, 2),
+    proposalStatus: withDefault(
+      NumberParam,
+      featureContext.featuresMap.get(FeatureId.PROPOSAL_FILTER)?.isEnabled
```

Comment:
Could a setting be better than a feature here? The setting could define `DEFAULT_INST_SCI_STATUS_FILTER` and map properly to the names.